docs/reference/esql/functions/date_format.asciidoc

```diff
@@ -1,12 +1,35 @@
 [discrete]
 [[esql-date_format]]
 === `DATE_FORMAT`
-Returns a string representation of a date in the provided format. If no format
-is specified, the `yyyy-MM-dd'T'HH:mm:ss.SSSZ` format is used.
+
+*Syntax*
 
 [source,esql]
 ----
-FROM employees
-| KEEP first_name, last_name, hire_date
-| EVAL hired = DATE_FORMAT("YYYY-MM-dd", hire_date)
+DATE_FORMAT([format,] date)
+----
+
+*Parameters*
+
+`format`::
+Date format (optional).  If no format is specified, the
+`yyyy-MM-dd'T'HH:mm:ss.SSSZ` format is used.  If `null`, the function returns
+`null`.
+
+`date`::
+Date expression. If `null`, the function returns `null`.
+
+*Description*
+
+Returns a string representation of a date in the provided format.
```

Comment:
```suggestion
Returns a string representation of a date, in the provided format.
```
Optional `,`, to suggest that the format applies to the output, not a characteristic of the date.